src/analysis/ffi_type.rs

```diff
@@ -73,7 +73,16 @@ fn ffi_inner(env: &Env, tid: TypeId, inner: &str) -> Result {
             use crate::library::Fundamental::*;
             let inner = match fund {
                 None => "libc::c_void",
-                Boolean => return Ok("glib_sys::gboolean".into()),
+                Boolean => {
+                    return Ok(format!(
+                        "{}ffi::gboolean",
+                        if env.library.is_glib_crate() {
+                            ""
```

Comment:
`crate::` if you want to make it more explicit, and we could have a helper function somewhere that returns `glib::` or `crate::` depending on `is_glib_crate()`. That should simplify this and a few other places.